src/app/game/settings/account/change/email/page.tsx

```diff
@@ -0,0 +1,72 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import NavigationFooter from "@/features/game/NavigationFooter";
+import React from "react";
+import { ChevronLeft } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+export default function GameSettingsAccountDelete() {
+  const pageTitle = "設定";
+  const pageSubTitle = "メールアドレス変更";
+
+  return (
+    <div className="relative h-full">
+      <h1 className="fixed inset-x-0 top-0 bg-[#0094f4] p-4 pt-7 text-center text-3xl text-white">
+        {pageTitle}
+      </h1>
+
+      <div className="pb-16 pt-20">
+        <div className="relative mx-6 flex h-[80px] items-center justify-around">
+          <div className="absolute left-0">
+            <Link href="/game/settings" className="p-0">
+              <ChevronLeft color="#000000" size={50} />
+            </Link>
+          </div>
+          <h1 className="text-2xl">{pageSubTitle}</h1>
+        </div>
+      </div>
+
+      <div className="mx-auto w-4/5">
+        <p className="mb-2 text-xl">確認のため、<span className="font-bold">パスワード</span>を入力してください。</p>
```

Comment:
重箱の隅をつつくようなものですが、
この場合は`span`よりも`strong`を使うほうが適切に思います！
他の部分も、該当するものは修正お願いします。
```suggestion
        <p className="mb-2 text-xl">確認のため、<strong className="font-bold">パスワード</strong>を入力してください。</p>
```

Reply:
@yoshiyuki-140 
調べたところ、strongよりもbタグの方が適切なようでがいかがでしょうか？
https://zero-plus.io/media/strong-b-br-span/